recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "bokeh" %}
-{% set version = "3.0.3" %}
+{% set version = "3.1.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/bokeh-{{ version }}.tar.gz
-  sha256: 1c28471ef5e6110ba5bed513137fd26054ebc4454bc768650eaeefc53b898a8a
+  sha256: 9047dfe50a671b5e19ca588fd90a8e6bff197114a5dbd6a59cc2678942c27887
 
 build:
   number: 0
```

Comment:
`xyzservices` is already available on s390x so the `skip` section can be modified

Reply:
numpy should be added to host:
`- numpy {{ numpy }} `
and in run modified as:
- {{ pin_compatible('numpy') }}

See also updated the lower version on numpy https://github.com/bokeh/bokeh/blob/5208b5fa8e8d6f3075a2c8072df81d86eff237e2/pyproject.toml#L14

Reply:
done